README.md

```diff
@@ -23,6 +23,7 @@ It provides:
 - sophisticated (broadcasting) functions
 - tools for integrating C/C++ and Fortran code
 - useful linear algebra, Fourier transform, and random number capabilities
+Besides its obvious scientific uses, NumPy can also be used as an efficient multi-dimensional container of generic data. 
```

Comment:
Lines < 80 characters please. This seems out of place and redundant to me, maybe use "array container object" above, or some such.

Reply:
Agreed, I don't think this adds anything given the mention of "ND array object" a few lines above.